content/11_SchedulingAndThreading.md

```diff
@@ -425,16 +425,32 @@ Although this scheduler is useful if you would prefer work not to happen on your
 
 The `EventLoopScheduler` provides one-at-a-time scheduling, queuing up newly scheduled work items. This is similar to how the `CurrentThreadScheduler` operates if you use it from just one thread. The difference is that `EventLoopScheduler` creates a dedicated thread for this work instead of using whatever thread you happen to schedule the work from.
 
-### HistoricalScheduler
+Unlike the schedulers we've examined so far, there is no static property for obtaining an `EventLoopScheduler`. That's because each one has its own thread, so you need to create one explicitly. It offers two constructors:
 
-The `HistoricalScheduler`. Should we do this one?
+```cs
+public EventLoopScheduler()
+public EventLoopScheduler(Func&lt;ThreadStart, Thread> threadFactory)
```

Comment:
Why is there an `&lt;` in the code fence?

Reply:
Because Lee typed it that way and I didn't notice.